src/components/MetricsOptions/MetricsOptionsBar.tsx

```diff
@@ -122,22 +108,27 @@ export class MetricsOptionsBar extends React.Component<Props, MetricsOptionsStat
     return (
       <Toolbar>
         {this.groupByLabelsHelper.renderDropdown()}
-        <div className="form-group">
-          <DropdownButton id="duration" title="Duration" onSelect={this.onDurationChanged}>
-            {MetricsOptionsBar.Durations.map(r => (
-              <MenuItem key={r[0]} active={r[0] === this.state.duration} eventKey={r[0]}>
-                {r[1]}
-              </MenuItem>
-            ))}
-          </DropdownButton>
-          <DropdownButton id="pollInterval" title="Polling interval" onSelect={this.onPollIntervalChanged}>
-            {MetricsOptionsBar.PollIntervals.map(r => (
-              <MenuItem key={r[0]} active={r[0] === this.state.pollInterval} eventKey={r[0]}>
-                {r[1]}
-              </MenuItem>
-            ))}
-          </DropdownButton>
-        </div>
+        <ToolbarDropdown
```

Comment:
Just to document, I didn't see in my previous review that there is a change in the component DropdownButton -> ToolbarDropdown. I guess it's same component but optimized for the Toolbar layout ?
I looks a good change, does it has any difference between the previous one ?

Reply:
Yes, ToolbarDropdown is a optimized component used in Graphtoolbar too, it's so similar for MetricOptions toolbar so I changed it, now it's cleaner